tsconfig.json

```diff
@@ -0,0 +1,42 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "ES2020",
+    "allowJs": true,
+    "checkJs": false,
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "outDir": "./dist",
+    "rootDir": "./",
+    "strict": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "alwaysStrict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": false,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "noUncheckedIndexedAccess": true,
+    //    "noPropertyAccessFromIndexSignature": false, // used by trigger output strings
+    "moduleResolution": "node",
+    "baseUrl": "./",
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true
+  },
+  "include": [
+    "./ui",
+    "./types",
+    "./resources",
+    "./util",
+    "./test"
+  ],
+  "exclude": [
+    "./eslint",
+    "./.eslintrc.cjs",
+    "./webpack",
+    "./dist",
+    "**/*.spec.ts"
```

Comment:
What is `*.spec.ts` use for by the way?

Reply:
tests(which haven't been written)

Reply:
![image](https://user-images.githubusercontent.com/19927330/107867374-6eae8880-6eb5-11eb-8700-f3082666d069.png)